### ISSUE TITLE
polymer:el fails when tests commented out.

### ISSUE THREAD

**Issue by wintersandroid** [yeoman/generator-polymer#276]
If you comment out tests in test/index.html the element generator will fail to create any files.

Took me a while to figure out that the tests I had commented out were the issue..

**Comment by robdodson** [comment]
Yeah there's a block in there that does a gnarly regex to try to auto-insert your tests for you. It's probably silently blowing up
